src/dev/customapp.md

```diff
@@ -1,11 +1,9 @@
 # Custom Mobile App
 
-The <MobileAppName /> provides seamless synchronisation of data between your phone/tablet and <MainPlatformName /> service. We know that sometimes people want a mobile app optimised for a specific use case rather than just using a general purpose app like <MobileAppName />.
+Our mobile app <MobileAppName /> provides seamless synchronisation of field data between mobile devices and <MainPlatformName /> service. However, we know that sometimes people want a mobile app optimised for a specific use case: maybe you just want to change the branding or disable some functionality - or maybe you want a bigger overhaul of the user interface. 
 
-The good news is that <MobileAppName /> is licensed as open source software and therefore it is possible to build custom apps off it. Maybe you just want to change the branding or disable some functionality - or maybe you want a bigger overhaul of the user interface - anything is possible with a bit of development. 
+The good news is that <MobileAppName /> is licensed as open source software, which means that anything is possible with a bit of development!
 
-## GitHub
+If you are interested in building a custom app, you can start by going to <GitHubRepo id="MerginMaps/input"/> repository on GitHub and building it for your platform. Here, you will find the <MobileAppName /> source code as well as the <GitHubRepo id="MerginMaps/input/blob/master/docs/developers/README.md" desc="developer documentation" />.
```

Comment:
I would remove "as well as the <GitHubRepo id="MerginMaps/input/blob/master/docs/developers/README.md" desc="developer documentation" />." that is not stable link and we can remove it soon